### ISSUE TITLE
doc: broken interop.rst

### ISSUE THREAD

**Issue by LeeTibbert** [scala-native/scala-native#3776]
Looks like recent changes to "interop.rst" broke it. Life happens.

Doc build log file from some WIP to fix Issue ##2687.  

There are two additional errors in the log file. I think they are cascade errors after the first.

I'll try to take a run at it/them after I finish my current work. 

```
/mnt/scala_native/2024/PR_doc_makedocs_I2687_2024-02-16T1323-0500/docs/user/interop.rst:132: ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 15 supplied.

.. code-block:: C
  int vprintf(const char * format, va_list arg);
  int printf(const char * format, ... );
```

**Comment by ekrich** [comment]
This doesn't fix the `makedocs` issue (I had to `pip install sphinx-last-updated-by-git`) but makes the code generate so maybe the other can be a separate PR. https://github.com/scala-native/scala-native/pull/3777

**Comment by LeeTibbert** [comment]
Looks like I did not get PR #3778 finished quickly enough to help you.  The intent was there.

**Comment by LeeTibbert** [comment]
I believe that Ekrich's now merged PR #3777 fixes this.  Thanks Eric.
